packages/app/pages/index.tsx

```diff
@@ -1,29 +1,59 @@
-import { User } from '@koh/common'
 import Router from 'next/router'
 import { ReactElement } from 'react'
-import { useHomePageRedirect } from '../hooks/useHomePageRedirect'
-import { useProfile } from '../hooks/useProfile'
-import { Spin } from 'antd'
+import { Button, Image, List } from 'antd'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  width: 500px;
+  padding-top: 100px;
+`
 
 export default function Home(): ReactElement {
-  const profile: User = useProfile()
-  const didRedirect = useHomePageRedirect()
-  if (profile && !didRedirect) {
-    Router.push('/courses')
-  } else {
-    return (
+  const login = () => {
+    Router.push('/login')
+  }
+
+  return (
+    <Container>
       <div
         style={{
-          position: 'absolute',
-          left: '50%',
-          top: '50%',
-          transform: 'translate(-50%, -50%)',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
         }}
       >
-        <Spin tip="Loading..." size="large" />
+        <img
+          src="/ubc_logo.png"
+          alt="UBC logo"
+          style={{ width: '3em', margin: '0 1em 0 0' }}
+        />
+        <h1 style={{ display: 'block', alignItems: 'center' }}>
+          Welcome to HelpMe
+        </h1>
       </div>
-    )
-  }
 
-  return <div />
+      <p style={{ margin: '15px' }}>
+        HelpMe Course System
+        <ul style={{ listStyleType: 'circle', textAlign: 'left' }}>
+          <li style={{ margin: '10px 0' }}>
+            Supports in-person and virtual office hours with instructors and
+            teaching assistants
+          </li>
+          <li style={{ margin: '10px 0' }}>
+            Chatbot for real-time answers about course content and course
+            questions
+          </li>
+          <li style={{ margin: '10px 0' }}>
+            For more information, contact Ramon Lawrence, ramon.lawrence@ubc.ca.
+          </li>
+        </ul>
+      </p>
+      <Button type="primary" onClick={login}>
```

Comment:
I feel like this shouldn't be a button, but link? i.e <a></a>

Reply:
I like the look of the button since it really sticks out on the page but i'll make a version with a link in a bit and we can compare on discord. also I realised I used a silly onclick router thing instead of just an href